src/cider/nrepl/middleware/complete.clj

```diff
@@ -36,7 +36,8 @@
 
 (def cljs-sources
   "A list of ClojureScript completion sources for compliment."
-  [::suitable-sources/cljs-source])
+  [::suitable-sources/cljs-source
+   :compliment.sources.local-bindings/local-bindings])
```

Comment:
Basically when I saw this line my first reaction was "how can this possibly work?". 😆

Reply:
Comment added (and PR updated)